src/components/interfaces/MOBILE_API.xml

```diff
@@ -1420,6 +1445,284 @@
+    </param>
+    <param name="PTY" type="Integer" minvalue="0" maxvalue="31" mandatory="false" >
+      <description>The program type - The region should be used to differentiate between EU and North America program types</description>
+    </param>
+    <param name="TP" type="Boolean" mandatory="false" >
+      <description>Traffic Program Identification - Identifies a station that offers traffic</description>
+    </param>
+    <param name="TA" type="Boolean" mandatory="false" >
+      <description>Traffic Announcement Identification - Indicates an ongoing traffic announcement</description>
+    </param>
+    <param name="REG" type="String" mandatory="false" >
+      <description>Region</description>
+    </param>
+  </struct>
+
+  <enum name="RadioState">
+    <element name="ACQUIRING"/>
+    <element name="ACQUIRED"/>
+    <element name="MULTICAST"/>
+    <element name="NOT_FOUND"/>
+  </enum>
+
+  <struct name="RadioControlData">
+    <param name="frequencyInteger" type="Integer" minvalue="0" maxvalue="1710" mandatory="false">
+      <description>The integer part of the frequency ie for 101.7 this value should be 101</description>
+    </param>
+    <param name="frequencyFraction" type="Integer" minvalue="0" maxvalue="9" mandatory="false">
+      <description>The fractional part of the frequency for 101.7 is 7</description>
+    </param>
+    <param name="band" type="RadioBand" mandatory="false">
+    </param>
+    <param name="rdsData" type="RdsData" mandatory="false">
+      <description> Read only parameter.</description>
+    </param>
+    <param name="availableHDs" type="Integer" minvalue="1" maxvalue="3" mandatory="false">
+      <description>number of HD sub-channels if available. Read only parameter.</description>
+    </param>
+    <param name="hdChannel" type="Integer" minvalue="1" maxvalue="3" mandatory="false">
+      <description>Current HD sub-channel if available</description>
+    </param>
+    <param name="signalStrength" type="Integer" minvalue="0" maxvalue="100" mandatory="false">
+      <description> Read only parameter.</description>
+    </param>
+    <param name="signalChangeThreshold" type="Integer" minvalue="0" maxvalue="100" mandatory="false">
+      <description>If the signal strength falls below the set value for this parameter,
+                   the radio will tune to an alternative frequency. Read only parameter.</description>
+    </param>
+    <param name="radioEnable" type="Boolean" mandatory="false">
+      <description> True if the radio is on, false is the radio is off.</description>
+    </param>
+    <param name="state" type="RadioState" mandatory="false">
+      <description> Read only parameter.</description>
+    </param>
+  </struct>
+
+  <struct name="RadioControlCapabilities">
+    <description>Contains information about a radio control module's capabilities.</description>
+    <param name="moduleName" type="String" maxlength="50" mandatory="true" >

```

Comment:
This param is not in the accepted proposal. Also why would it be mandatory?

Reply:
@joeygrover module name is not same parameter as others in capabilities, it is short description, and according to https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0071-remote-control-baseline.md it should be mandatory. Do you think it shouldn't?